source/config.js

```diff
@@ -1,5 +1,16 @@
+
+const DEFAULT_API_CLIENT_ID = 'ae51d45d93313f1';
+
+function getApiClientId() {
+    let apiClientId = process.env.IMGUR_CLIENT_ID;
+    if (!apiClientId) {
+        apiClientId = DEFAULT_API_CLIENT_ID;
+    }
+    return apiClientId;
+}
+
 export default {
-    API_CLIENT_ID: 'ae51d45d93313f1',
+    API_CLIENT_ID: getApiClientId(),
```

Comment:
@ReallyLiri Thanks for the quick change! May I suggest a more concise way of writing this. This can also be written as:

```js
API_CLIENT_ID: process.env.IMGUR_CLIENT_ID || DEFAULT_API_CLIENT_ID
```

If `IMGUR_CLIENT_ID` env variable is not defined, `process.env.IMGUR_CLIENT_ID` will be `undefined` and the code will fallback to the `DEFAULT_API_CLIENT_ID`. It's the same idea as the `getApiClientId` function 😉

Reply:
very nice, I was not familiar with this syntax in JS :)

Reply:
Awesome, we all learn something new everyday! And this PR looks great to me now 💯 Let me merge and create a new release for this package. Thank you very much! 🎉